src/elements/inputGroup/InputGroup.tsx

```diff
@@ -19,12 +31,23 @@ const InputGroup: FC<Props> = ({
       )}
       <input
         type="text"
-        className={classnames('form-control', {
-          borderRadiusLeft: !prepend,
-          borderRadiusRight: !append,
-        })}
-        placeholder={inputPlaceholder}
+        value={value}
+        id={id}
```

Comment:
We also need a prop "name" for this input please.

Reply:
Done 👍